test/recipes/30-test_evp_data/evpkdf.txt

```diff
@@ -254,51 +254,51 @@ Output = 2c91117204d745f3500d636a62f64f0ab3bae548aa53d423b0d1f27ebba6f5e5673a081
 
 Title = id-scrypt tests (from draft-josefsson-id-scrypt-kdf-03 and others)
 
-KDF = scrypt
+KDF = id-scrypt
```

Comment:
Ouch

Reply:
This is why aliases would be good (for all of these id- cases)

Reply:
So that would be because the old code looked for both SN and LN I guess.

Reply:
Yes, I totally agree, and I have two alternative PRs doing this waiting for resolution (they do need an update, that goes without saying, but the silence is pretty stunning).  #8984, #8985 (they are both WIP because they are alternative for each other, *and a choice has to be made*)

Reply:
Allowing the long name would resolve this.  It seems that only the SN works at the moment.

I do agree that this is sucky.

Reply:
> Allowing the long name would resolve this.

Meaning what?  We must still add it *somewhere*, and we've already established that we don't want to use the old object database because of its fairly tight relationship with ASN.1 (plus it's a bit limited)

Reply:
BTW, this isn't really the best place for that discussion...

Reply:
Agreed, elsewhere and perhaps later.